protocol.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/cloudflare/cfssl/helpers"
 )
 
+//go:generate stringer -type=Tag,Op -output=protocol_string.go
```

Comment:
do I need to be extra careful with 'go generate' inside the vendored code?
In other words, can we checked in the ugly generated protocol_string.go so I don't need to worry about it?